nbstripout.py

```diff
@@ -161,7 +161,7 @@ def install(attrfile=None):
     except CalledProcessError:
         print('Installation failed: not a git repository!', file=sys.stderr)
         sys.exit(1)
-    check_call(['git', 'config', 'filter.nbstripout.clean', '%s %s' %
+    check_call(['git', 'config', 'filter.nbstripout.clean', '\\"%s\\" \\"%s\\"' %
```

Comment:
Why do you escape the `"`? Does this actually work on Windows? It fails on Linux:
```
\"/usr/bin/python\" \"/path/to/nbstripout/nbstripout.pyc\": 1: \"/usr/bin/python\" \"/path/to/nbstripout/nbstripout.pyc\"
error: external filter \"/usr/bin/python\" \"/path/to/nbstripout/nbstripout.pyc\" failed
fatal: test.ipynb: clean filter 'nbstripout' failed
```